app/include/zmk/hid.h

```diff
@@ -137,6 +137,21 @@ static const uint8_t zmk_hid_report_desc[] = {
     0x00,
     /* END COLLECTION */
     HID_MI_COLLECTION_END,
+
+#if IS_ENABLED(CONFIG_ZMK_PLOVER_HID)
+    0x06, 0x50, 0xff,              // UsagePage (65360)
```

Comment:
Please use the defines provided by Zephyr for this, not these bare values. See the example code above, or the Zaphyr in `includes/usb/class/usb_hid.h`.

Reply:
Sure, I can switch to using the macros. A suggestion for the future is to look up the [hid resource descriptor compiler](https://github.com/nipo/hrdc) that I used to generate the descriptor. That makes the whole job of creating correct descriptors a lot easier, but for now I'm ok with switching to the macros.